### ISSUE TITLE
Where can I configure the redis host and port?

### ISSUE THREAD

**Issue by nicolasparada** [ndugger/blackbeard#1]
In dev, I use docker for things like databases, so redis isn't `localhost:6379` and I can't even start blackbeard :(

**Comment by ndugger** [comment]
Honestly, I forgot to add in the redis configurations in the setup; I'll add this in first thing tomorrow, and keep you updated!

Sorry about that!

**Comment by ndugger** [comment]
I've published v0.1.1-beta to npm that includes a fix to the setup script that will now prompt you for Redis configurations.

Go ahead and grab the latest version from npm:
- change your version # in package.json to **0.1.1-beta**
- `npm install blackbeard`

However, since you've already ran the setup, running it again would be silly, so open up your "blackbeard.config" JSON file, and replace the "server" property with the following structure (obviously edit the data)

``` json
  "server": {
    "port": "3000",
    "redis": {
      "enabled": true,
      "defaultMaxAge": 60,
      "host": "localhost",
      "port": "6379",
      "username": "",
      "password": ""
    }
  },
```

**Comment by nicolasparada** [comment]
Yeah, working fine now. Keep the great work @ndugger :)
